cachelib/cachebench/runner/AsyncCacheStressor.h

```diff
@@ -303,7 +304,14 @@ class AsyncCacheStressor : public Stressor {
       } else {
         wHdl = std::move(hdl).toWriteHandle();
       }
+#if defined(__GNUC__) && !defined(__clang__) && __GNUC__ == 8
+      assert(req->sizeBegin + 1 != req->sizeEnd);
+#else
+      // gcc-8.5 crashes with an internal compiler error at typeck.c:5965
+      // here. Suspected regression in folly as this line compiled fine
+      // with 8.5 previously.
       XDCHECK(req->sizeBegin + 1 != req->sizeEnd);
+#endif
```

Comment:
It is unfortunate that there is some regression for XDCHECK, but this looks like an overkill. Can you simply move this right before the lambda?

Reply:
Good catch. Done, thanks!